enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/Editor.tsx

```diff
@@ -22,7 +22,7 @@ export const Editor = ({ onApply = () => {} }: EditorProps) => {
     question,
     originalQuestion,
     updateQuestion,
-    runQuestion,
+    queryQuestion,
```

Comment:
you'll see where this is renamed later on.

Bascially, I want to make it clear which function does what. There's a `queryQuestion` and `loadAndQueryQuestion`.

The first one only query existing question, but the later load a question first then query it.